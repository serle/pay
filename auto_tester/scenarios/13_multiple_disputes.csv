type,client,tx,amount
deposit,1,1,100.0
deposit,1,2,50.0
deposit,1,3,25.0
dispute,1,1,
dispute,1,2,
resolve,1,1,
chargeback,1,2,
