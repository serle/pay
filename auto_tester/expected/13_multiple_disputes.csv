client,available,held,total,locked
1,125.0000,0.0000,125.0000,true
